backend/src/services/org/org-role-fns.ts

```diff
@@ -0,0 +1,56 @@
+import { OrgMembershipRole } from "@app/db/schemas";
+import { TFeatureSet } from "@app/ee/services/license/license-types";
+import { BadRequestError } from "@app/lib/errors";
+import { TOrgRoleDALFactory } from "@app/services/org/org-role-dal";
+
+const RESERVED_ORG_ROLES = Object.values(OrgMembershipRole).filter((role) => role !== "custom");
+
+// this is only for updating an org
+export const getDefaultOrgMembershipRoleForUpdateOrg = async ({
```

Comment:
I think this is a bit confusing that it can return both a slug and an ID. Can we somehow normalize the return type? An example would be to return a full role object, or return an object with `{roleSlug, and customRoleId}`?

Reply:
That's what the respective `col` is though on the table, either an ID or Slug, I specified slug when it is exclusively slug; this function is only ever used when directly assigning to the `col` `defaultMembershipRole`, so returning an object would just needlessly complicate the assignment requiring a ternary for a check that's already been done